trade-finance-manager-ui/server/controllers/case/index.js

```diff
@@ -76,7 +76,7 @@ const getCaseDeal = async (req, res) => {
       tfm: deal.tfm,
       successMessage,
       activePrimaryNavigation: 'manage work',
-      activeSubNavigation: 'deal',
+      activeSubNavigation: 'deals',
```

Comment:
I missed these.  Can be done in the next PR unless you want to do it now?

Reply:
Pushed